omegatrade-app/frontend/src/app/components/stock-dashboard/stock-dashboard.component.ts

```diff
@@ -0,0 +1,182 @@
+import { Component, OnInit } from '@angular/core';
+import { Chart, StockChart } from 'angular-highcharts';
+import { ActivatedRoute } from '@angular/router';
+import { RestService } from '../../services/rest.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { interval, Subscription } from "rxjs";
+import { take } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-stock-dashboard',
+  templateUrl: './stock-dashboard.component.html',
+  styleUrls: ['./stock-dashboard.component.css']
+})
+export class StockDashboardComponent implements OnInit {
+
+  selectedCompany: string = "";
+  lastUpdatedTime;
+  stock: StockChart;
+  stocks: any;
+  company: any;
+  companies: any;
+  loader: boolean = false;
+  subscription: Subscription;
```

Comment:
`subscription: Subscription;`
This subscription variable we can rename it like `updatePointIntervalSubscription` or something related instead of generic names

Reply:
This Subscription is removed as it is not required.